scripts/uberenv/spack_configs/darwin/packages.yaml

```diff
@@ -27,7 +27,7 @@ packages:
       lapack: [openblas]
       mkl: [intel-mkl]
       mpe: [mpe2]
-      mpi: [mpich, openmpi]
+      mpi: [openmpi, mpich]
```

Comment:
My system does not like the mpich patch

Reply:
I added a bug report for the patch issue with mpich 3.2.1:
https://github.com/spack/spack/issues/8432

I am having bad luck with openmpi -- I suggest we switch back and prefer another version of mpich - 3.2 (sans .1) which will patch, or even older 3.0.4 should work.